calculate_average_rmannibucau.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+#
+#  Copyright 2024 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# sdk use 21-zulu
+
+# $script build -> build on 21-zulu before running
+# $script native -> use native-image and build before running
+
+function run() {
+  JAVA_OPTS="--enable-preview"
+  java $JAVA_OPTS --class-path target/classes dev.morling.onebrc.CalculateAverage_rmannibucau
+}
+
+if [ -n "$1" ]; then
```

Comment:
So do you want to submit this as native or JVM? If native, please extract the build step into a separate script. See Thomas Würthinger's implementation for an example.

Reply:
jvm, graalvm boost is not impressive enough to be relevant IMHO - it is literally in the error zone - (was some ms on my tests/computer, just kept it in the script to enable it and potentially tune it more but default run is really what should be evaluated)